steampy/client.py

```diff
@@ -416,3 +416,39 @@ def get_wallet_balance(self, convert_to_decimal: bool = True, on_hold: bool = Fa
             return Decimal(balance_dict[balance_dict_key]) / 100
         else:
             return balance_dict[balance_dict_key]
+
+    @login_required
+    def add_friend(self, steam_id: str) -> bool:
+        response = self._friend_ajax_request(steam_id)
+        return True if response else False
+
+    @login_required
+    def accept_friend(self, steam_id: str) -> bool:
```

Comment:
what is the difference between add_friend and accept_friend? seems like the only diffrerence is checking the response